codecov.yml

```diff
@@ -7,6 +7,11 @@ coverage:
       default: false
     patch:
       default: false
+  ignore:
+    - "Externals/*"
```

Comment:
💭 This one is debatable. I don't see particular value in ignoring it right now.

Reply:
Do we run tests for these projects as part of the GE build? Shouldn't that be the job of CI for those projects, separately? I consider `Externals` as libraries, and therefore not the consuming project's responsibility to treat as a test subject directly.

Reply:
The overall coverage number is unrelated to project quality, and we gain nothing by excluding coverage from a portion of our source build. The only case where this would be an advantage is the following:

1. The coverage information for the external is broken (e.g. the current exclusions)
2. The coverage information for the external causes problematic performance overhead during the build

On the other hand, retaining the coverage information allows us to better understand the portions of these external libraries we use. This information can provide supporting evidence in risk assessment when pulling changes from the upstream repository.

Reply:
For myself if I want information on library usage, I use local tools that show coverage/diagrams and so forth. I hadn't considered useing the codecov output. Taking a look through just now I, would still use my own tooling, but if it's useful to others then we can leave it. Removed from the PR.

Reply:
If it counts, in the short term I don't see a great deal of value in the code coverage for the external dependencies. For me they are almost like inescapable evil we have to suffer....
In the long term though @sharwell has a point.